spec/factories/item.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :item do
+    name        "Salted Caramel Peanut Butter Cup"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.